auth-web/src/components/auth/staff/continuation-application/ContinuationApplicationTable.vue

```diff
@@ -68,7 +163,12 @@
                       v-for="(header, i) in headers"
                       :key="getIndexedTag('find-header-row', i)"
                       :scope="getIndexedTag('find-header-col', i)"
-                      class="font-weight-bold"
+                      :class="{
+                        'font-weight-bold': true,
```

Comment:
No need to change this but FYI: since this class is always set, you can use the `class` prop on this element. (Ie, you can use both `class` and `:class`.)